views/partials/subpages/confirm.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="dashboard-email-sent bold"> {{ getString "email-verified" }}</span>
+<h1>{{ getString "email-added-to-subscription" email=email }}</h1>
+{{{ getString "email-verified-view-dashboard" sign-in-link='<a href="/oauth/init">Sign in</a>'}}}
```

Comment:
Sorry to dogpile this line, but, I think the "Sign in" should be lowercased, per @betsymi's https://github.com/mozilla/blurts-server/issues/1087#issuecomment-534675956

> Email Successfully Verified!
>
> We’ll alert you if emailaddress@email.com appears in a data breach. To see and manage all emails you’ve signed up for breach monitoring, [sign in](#).